plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/manager/VmwareStorageManagerImpl.java

```diff
@@ -1094,13 +1092,6 @@ private String createOVAFromMetafile(String metafileName) throws Exception {
         } catch (Exception e) {
             s_logger.error("Exception while creating OVA using Metafile", e);
             throw e;
-        } finally {
-            if (strm != null) {
-                try {
-                    strm.close();
```

Comment:
Since you removed the finally block, the FileInputStream is not getting closed. Put it at the end of the try block at least :)